sql-cli/sql_cli/cli.py

```diff
@@ -14,11 +17,16 @@
 from sql_cli.project import Project
 
 
-def generate_dag(project: Project, workflow_name: str) -> Path:
+def generate_dag(project: Project, env: str, workflow_name: str, generate_tasks: bool = False) -> Path:
+    rprint(
+        f"\nGenerating the DAG file from workflow [bold blue]{workflow_name}[/bold blue]"
+        f" for [bold]{env}[/bold] environment..\n"
+    )
```

Comment:
@dimberman I think there went something wrong on merging main into your branch. With that change we now have duplicate print statements in generate.